lib/heuristics/scheduling_heuristic.rb

```diff
@@ -923,14 +923,10 @@ def get_unassigned_info(custom_id, service_in_vrp, reason)
         original_service_id: service_in_vrp.id,
         service_id: custom_id,
         point_id: service_in_vrp.activity&.point_id,
-        detail: {
-          lat: service_in_vrp.activity&.point&.location&.lat,
-          lon: service_in_vrp.activity&.point&.location&.lon,
-          setup_duration: service_in_vrp.activity&.setup_duration,
-          duration: service_in_vrp.activity&.duration,
-          timewindows: service_in_vrp.activity&.timewindows ? service_in_vrp.activity.timewindows.collect{ |tw| { start: tw.start, end: tw.end } }.sort_by{ |t| t[:start] } : [],
-          quantities: service_in_vrp.quantities.collect{ |qte| { unit: qte.unit.id, value: qte.value, label: qte.unit.label } }
-        },
+        detail: Wrappers::Wrapper.new.build_detail(service_in_vrp,
```

Comment:
There are more `Wrappers::Wrapper.new.build_detail` calls below, I don't mark them to not to clutter the review but we should do the same for them as well.

Reply:
hmmm, annoying indeed :/ and wasteful but I guess we need to make do with what we have at the moment.. so until we refactorize the whole "algorithms" logic, this looks like the only way. 

One "hack" might be to create a "wrapper" object in the scheduling class once at the beginning, so that whenever you need it you can do `@wrapper.build_detail` inside scheduling class? But it looks like to me that this is a bit outside the scope of this PR, so I leave you to decide what's best.

Reply:
I can do this in a new PR. In fact I am also creating a new Wrapper object to generate empty_result.
We already have a lot of global variables but it might be better with this additional (temporary one).

Reply:
cool. just to be clear, i didn't suggest a global variable but a class instance variable for the `Scheduling` class.

Reply:
https://github.com/Mapotempo/optimizer-api/pull/132